### PR TITLE
LCSEC-359 add a hosting service with an image from nondockerio registry

### PR DESCRIPTION
**Pull request by yijutuan**

### CODE REVIEW

File: nondocker-internal-service-image/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# hosting
```
Comment (christieyoo) [inline nondocker-internal-service-image/README.md:1]: Could you replace this line with `nondocker-internal-service-image`?

**Review by christieyoo — CHANGES_REQUESTED** [review]
Just requesting one change. Other than that, LGTM. Thanks for adding the README too :)

**Review by emanuelfisilva — APPROVED** [review]
Thanks for working on this @yijutuan !
LGTM

File: nondocker-internal-service-image/README.md (hunk 1 above)
Comment (yijutuan) [reply nondocker-internal-service-image/README.md:1]: oops forgot to change that 😆

**Review by anthony-chu — APPROVED** [review]
lgtm!

**Review by tainguyen100 — APPROVED** [review]
LGTM

**Pull request merged**